chapters/dae.tex

```diff
@@ -51,7 +51,7 @@ \chapter{Modelica DAE Representation}\doublelabel{modelica-dae-representation}
 \textbf{m}(t\textsubscript{e}) & Modelica variables of type
 \emph{discrete Real, Boolean, Integer} which are unknown. These
 variables change their value only at event instants t\textsubscript{e}.
-pre(m) are the values of m immediately before the current event
+\lstinline!pre!(m) are the values of m immediately before the current event
```

Comment:
I think the (m) should be part of the listinline here.

Reply:
The old document had exactly the formatting above, i.e. pre as code - but (m) and m as normal text, so I think we can leave it as is. Either it was a pure mistake, or the intention was that m is a variable in the pseudo-code and not in Modelica.

Reply:
Thank you for spotting it. It looked strange, so I fixed it.

There are more "taste" decisions and the LaTeX document isn't a one-to-one copy. It just should look similar.